frontend/src/components/listofproducts/AddNewProductButton.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Button } from 'react-bootstrap'
+import { useTranslation } from 'react-i18next'
+
+const AddNewProductButton = () => {
+  const { t } = useTranslation()
+  return (
+    <Link to="items/new-item">
```

Comment:
Ez szerintem relatívan fog hivatkozni, ami bizonyos oldalakon lehet, hogy problémát fog okozni.

Reply:
Értem, ez pont véletlenül jól működik, mert a "/super-admin/items" oldalon használjuk :) Reméljük, hogy nem fogjátok használni más helyen és reméljük, hogy nem fog változni a lista elérési útja mondjuk /super-admin/items/list-re. Vagy a remény helyett át is írhatnánk abszolút URL-re: "/super-admin/items/new-item" Ha ezt nem csak a szuperadmin használja, akkor meg kaphat esetleg egy baseURL-t prop-ban.